src/cdk-experimental/popover-edit/edit-events.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Subject, timer} from 'rxjs';
+import {audit, distinctUntilChanged, filter, map, share} from 'rxjs/operators';
+import {Injectable} from '@angular/core';
+import {closest, CELL_SELECTOR, DEFAULT_HOVER_DELAY_MS, ROW_SELECTOR} from './common';
+
+/**
+ * Service for sharing delegated events and state for triggering edits.
+ */
+@Injectable()
+export class EditEvents {
+  readonly editing = new Subject<Element|null>();
```

Comment:
Need comments on these variables describing them (same for other files/classes)

Reply:
Added comments for most fields (except in a couple very self-explanatory cases).